cpp/src/dictionary/decode.cu

```diff
@@ -30,6 +29,18 @@
 namespace cudf {
 namespace dictionary {
 namespace detail {
+namespace {
+struct indices_handler_fn {
```

Comment:
Thanks for fixing this.  Would there be value in naming this functor a "nullifying_handler" or some such?

Reply:
The actually nullifying happens far away in the gather function. This function simply returns indices.